source/includes/compass-executable.rst

```diff
@@ -0,0 +1,22 @@
+The name and location of the |compass-short| executable varies by
+operating system. 
+
+Linux
+~~~~~
+The executable is called ``mongodb-compass``. The installer installs it
+in the ``/usr/bin`` directory. 
+
+Windows
+~~~~~~~
+
+The executable is called ``MongoDBCompass.exe``. The installer installs
+it in a folder you pick during the installation process.
+
+MacOS
+~~~~~
+The executable is called ``Compass``. The installer installs
```

Comment:
I don't think this is quite correct. I think the executable is technically `./"MongoDB Compass"`.

For example, calling `Compass` does not work on my machine:

```
[joe! /Applications/MongoDB Compass/Contents/MacOS]$ ./"Compass"
-bash: ./Compass: No such file or directory
```

However, this opens Compass:

```
[joe! /Applications/MongoDB Compass/Contents/MacOS]$ ./"MongoDB Compass"
```

Reply:
Done.

Reply:
Thanks!